src/templates/sermon.scss

```diff
@@ -1,9 +1,20 @@
 .template-sermon {
-    h1 {
-        margin-bottom: 0.25rem;
+    .template-sermon__meta {
```

Comment:
If you're using BEM classnames do you want to move this style out of `.template-sermon` to avoid accumulating specificity?

Reply:
Since I try to keep almost all styling scoped, and styles fairly shallow within its top-level scope, I haven't run into a problem with this.